localization/camera/CMakeLists.txt

```diff
@@ -54,38 +50,41 @@ catkin_package(
 # Specify additional locations of header files
 include_directories(
   include
-  ${catkin_INCLUDE_DIRS}
   ${EIGEN3_INCLUDE_DIR}
   ${OpenCV_INCLUDE_DIRS}
+  ${LUAJIT_INCLUDE_DIR}
```

Comment:
do we know why these aren't brought in by the config_reader dependency?

Reply:
It complained about not finding it on a piece of code in the camera package that imported a lua header, so not through the config_reader. I'm not sure why it didn't get it through the config reader..

Reply:
you also need to add filesystem to the boost components that's probably why

Reply:
actually, sorry, we shouldn't need this at all. Could you move all the includes here to the .cpp file? https://github.com/nasa/astrobee/blob/ros2/shared/config_reader/include/config_reader/config_reader.h 

You can forward declare the lua_State struct

Reply:
ups, I deleted those from the cmake and it worked (compiled from scratch and ran the tests), I think some other PR might have fixed the linking problem